untrack/src/lib.rs

```diff
@@ -76,21 +72,51 @@ impl Args for Options {
 }
 
 pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
-    let (urn, _) =
+    let (urn, repo) =
         project::cwd().context("this command must be run in the context of a project")?;
     let profile = ctx.profile()?;
 
-    execute(&urn, options, &profile)
+    execute(&urn, Some(&repo), options, &profile)
 }
 
-pub fn execute(urn: &Urn, options: Options, profile: &Profile) -> anyhow::Result<()> {
-    // TODO: Remove remote
-    // TODO: Remove tracking branch
+fn get_peer_id(
+    project: &project::Metadata,
+    storage: &Storage,
+    name: &String,
+) -> anyhow::Result<Option<PeerId>> {
+    let mut found = None;
+    for (id, peer) in project::tracked(project, storage)? {
+        if peer.name() == *name {
+            found = Some(id);
+        }
+    }
+
+    Ok(found)
+}
 
+pub fn execute(
+    urn: &Urn,
+    repo: Option<&git::Repository>,
+    options: Options,
+    profile: &Profile,
+) -> anyhow::Result<()> {
     let signer = term::signer(profile)?;
     let storage = keys::storage(profile, signer)?;
 
-    if let Some(peer) = options.peer {
+    if let Some(peer_str) = options.peer {
+        let peer = if let Ok(val) = PeerId::from_str(&peer_str) {
+            val
+        } else {
+            let project = project::get(&storage, urn)?
+                .ok_or_else(|| anyhow!("project {} not found in local storage", &urn))?;
+
+            if let Some(v) = get_peer_id(&project, &storage, &peer_str)? {
+                v
+            } else {
+                anyhow::bail!("invalid <peer-id> '{}'", peer_str)
+            }
+        };
```

Comment:
Ok so this allows untracking a peer by name?

Reply:
Yes, I should have referenced this in the commit message